### ISSUE TITLE
[New Feature] implement a way to check if a module is remote

### ISSUE THREAD

**Issue by qku** [Ulm-IQO/qudi-core#64]
### Feature Description

Sometimes it's nice to know if a module (hardware mostly probably) is running on a remote qudi instance. Some methods may work differently or not at all through a remote connection. In order to know when to use alternatives, the module that is connected to the remote module should know if it is remote.

### Considered Alternatives

There may be ways to use the module manager to find out if a module is remote, but these are likely difficult to access. Another alternative is to check if objects passed back from a module are of the `netref` type. This requires the passing of some kind of test object though.

**Comment by Neverhorst** [comment]
Just to clarify:
You want to check the connection to a module for remote access, right? E.g. something like:
```Python
class MyLogicModule(LogicBase):
    _my_hardware_module = Connector(...)

    def do_stuff(self, ...):
        if _my_hardware_module.is_remote:
            ...
```

**Comment by qku** [comment]
Yes, I would like to check if a module is remote from a connected module. I guess it does not make sense for a module to test if itself is remote, as the remoteness only stems from the relationship between two modules.

**Comment by Neverhorst** [comment]
Exactly, that was my line of thought.
The checker on the `Connector` can surely be implemented and will probably come with the promised module management overhaul.
